go.mod

```diff
@@ -141,6 +141,7 @@ require (
 // github.com/operator-framework/operator-sdk.
 replace (
 	bitbucket.org/ww/goautoneg => github.com/munnerz/goautoneg v0.0.0-20120707110453-a547fc61f48d
-	github.com/openshift/api => github.com/openshift/api v0.0.0-20230607151152-bdd886567621
 	k8s.io/client-go => k8s.io/client-go v0.27.2
 )
+
+replace github.com/openshift/api => github.com/openshift/api v0.0.0-20230807132801-600991d550ac
```

Comment:
@miheer please fix this, it should stay within the replace block.

Reply:
OK I generally don't modify this file manually. With the go edit command it places it separately.

Reply:
however I can place it manually if you want that way.

Reply:
https://github.com/openshift/cluster-ingress-operator/compare/b7d596cbe45f5c7a621498f14036337633094395..5ceabd73033d098a2b3006370c1309b346ee1deb  done